Assets/Script/Menu/Main/MainMenu.cs

```diff
@@ -8,12 +9,14 @@
 using YARG.Menu.Navigation;
 using YARG.Menu.Persistent;
 using YARG.Settings;
+using Cysharp.Threading.Tasks;
 
 namespace YARG.Menu.Main
 {
     public class MainMenu : MonoBehaviour
     {
         private static bool _antiPiracyDialogShown;
+        private static bool _blurbPlayed;
 
```

Comment:
So you want to have the title blurb wait until after the anti-piracy dialog closes? Save a reference to the dialog, then await its `WaitUntilClosed` method in OnEnable.

Declare reference here:

```suggestion
        private static OneTimeMessageDialog _antiPiracyDialog;
```